README.md

```diff
@@ -1549,6 +1549,37 @@ Query `posts(limit:10,page:1){data{id},total,per_page}` might return
 Note that you need to add in the extra 'data' object when you request paginated resources as the returned data gives you
 the paginated resources in a data object at the same level as the returned pagination metadata.
 
+Simple Pagination will be used, if a query or mutation returns a `SimplePaginationType`.
+
+```php
+namespace App\GraphQL\Queries;
+
+use Closure;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Query;
+
+class PostsQuery extends Query
+{
+    public function type(): Type
+    {
+        return GraphQL::simplePaginate('posts');
```

Comment:
I think this can't be `null`? (Same for the existing example I would say, but let's stay focused on this PR)
```suggestion
        return Type::nonNull(GraphQL::simplePaginate('posts'));
```